lib/elixir/lib/exception.ex

```diff
@@ -2131,6 +2131,38 @@ defmodule UnicodeConversionError do
   end
 end
 
+defmodule MissingApplicationsError do
```

Comment:
Sorry, we have gone through some many bikesheds. I think last we talked it was meant to be singular? MissingApplicationError? I am fine either way, we should just double check.

Reply:
We agreed on plural IIRC, because we have a list of apps in the exception?

Reply:
I thought some found it weird because we don’t have many plurals in module names. Your call!

Reply:
Well it's still the singular `Error` so let's go with `MissingApplicationsError` 😄